### PR TITLE
Select dropdown

### PR DESCRIPTION
**Pull request by allisonking**
## Issue(s) Resolved

Closes https://github.com/pubpub/platform/issues/613

## High-level Explanation of PR
Renders the select dropdown

## Test Plan

* Add a numeric array or a string array field
* Add to a form as a select dropdown
* Open the form fill page. The select dropdown should render as expected and work to update/create a pub.

## Screenshots (if applicable)

https://github.com/user-attachments/assets/2b9a2c92-4d45-4875-9913-d41bbc16dde6

### CODE REVIEW

File: core/playwright/externalFormCreatePub.spec.ts [hunk 1]
```diff
@@ -70,11 +70,13 @@ test("Can create a pub from an external form", async () => {
 
 test.describe("Multivalue inputs", () => {
 	test("Can add a radio and checkbox multivalue input", async () => {
+		test.setTimeout(60_000);
```
Comment (allisonking) [inline core/playwright/externalFormCreatePub.spec.ts:73]: 🙈

Comment (3mcd) [reply core/playwright/externalFormCreatePub.spec.ts:73]: This is just overriding the maximum amount of time this test can run before failing right?

**Review by 3mcd — APPROVED** [review]
Nice!

File: core/playwright/externalFormCreatePub.spec.ts (hunk 1 above)
Comment (allisonking) [reply core/playwright/externalFormCreatePub.spec.ts:73]: yup that's right! default is 30s

**Pull request merged**